docs/Logging.md

```diff
@@ -0,0 +1,65 @@
+# Logging
+
+OTP uses [logback](http://logback.qos.ch/) and [slj4j](http://www.slf4j.org/) as a logging framework.
+Logging is configured in the `logback.xml` file inside the OTP jar file. See these frameworks for
+more documentation on log configuration.
+
+For developers, starting OTP using the `InteractiveOtpMain` is an easy way to configure debug
+logging.
+
+Some loggers useful for debugging.
+
+- `TRANSFERS_EXPORT`: Dump transfers to _transfers-debug.csv_ file.
+- `DATA_IMPORT_ISSUES`: Write issues to debug lag as well as to the issue report.
+- `REQ_LOG`: Router request log. Enable with `requestLogFile` config parameter in build config.
+- `org.opentripplanner.raptor.RaptorService`: Debug Raptor request and response
+
+## Format
+
+By default, OTP logs in plain text to the console. However, it is possible also log in JSON format.
+
+To enable it, set the Java property `otp.logging.format` to one of these values:
+
+- `plain`: regular plain text logging (default, no need to configure it)
+- `json`: Logstash-encoded JSON format understood by many log ingestion tools (Datadog, Loggly, Loki...)
+
+### Complete example
+
+```
+java -Dotp.logging.format=json -jar otp.jar --load --serve data
+```
+
+### Further customization
```

Comment:
I would mention here that you can override the logback.xml file embedded in the OTP jar with an external logback.xml file by setting the property -Dlogback.configurationFile (no need to manually edit the embedded logback.xml file)

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/5023/commits/ae7e5d0e735761e11eb622df86fc8bad69d94e31